sklearn/metrics/classification.py

```diff
@@ -1393,7 +1393,7 @@ class 2       1.00      0.67      0.80         3
         labels = np.asarray(labels)
 
     if target_names is not None and len(labels) != len(target_names):
-        warnings.warn("labels size, {}, does not match size of target_names, {}".format(len(labels), len(target_names))
```

Comment:
Thanks for the advice, I wasn't aware of these tools. I got normal flake8 working but I use pycharm mostly and can't find a plugin for it (and the built in inspector doesn't catch line length. Any advice on an editor/IDE I could use?